# Задача 1. Сделай предсказания с помощью дискретной модели


Чтобы устроиться на работу в IT-компанию, кандидат проходит несколько этапов собеседований. На первом этапе кандидат должен правильно решить не меньше 5 задач из 8, чтобы пройти на второй этап.

Сейчас компания опирается на представленную в таблице вероятностную модель для количества $X$ задач, которые кандидат правильно решил на первом этапе.

<table>
<caption>
Распределение $X$ — количества задач, правильно решенных кандидатом.
</caption>
  <thead>
    <tr>
      <th>$k$</th>
      <th>$0$</th>
      <th>$1$</th>
      <th>$2$</th>
      <th>$3$</th>
      <th>$4$</th>
      <th>$5$</th>
      <th>$6$</th>
      <th>$7$</th>
      <th>$8$</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>$P(X=k)$</b></td>
      <th>$0.23$</th>
      <th>$0.24$</th>
      <th>$0.12$</th>
      <th>$0.11$</th>
      <th>$0.04$</th>
      <th>$0.08$</th>
      <th>$0.10$</th>
      <th>$0.05$</th>
      <th>$0.03$</th>
    </tr>
  </tbody>
</table>



## 1. (blue, баллы: 0.4) Предскажи, с какой вероятностью кандидат пройдёт интервью.

$$
P(X >= 5) = P(X = 5) + P(X = 6) + P(X = 7) + P(X = 8) = 0.08 + 0.10 + 0.05 + 0.03 = 0.26
$$

## 2. (blue, баллы: 0.4) Найди вероятность, что кандидат решит от 2 до 6 задач включительно.

$$
P(2 <= X <= 6) = P(X = 2) + P(X = 3) + P(X = 4) + P(X = 5) + P(X = 6) = 0.12 + 0.11 + 0.04 + 0.08 + 0.10 = 0.45
$$

## 3. (blue, баллы: 0.4) Найди вероятность, что кандидат решит от 2 до 6 задач не включительно.

$$
P(2 < X < 6) = P(X = 3) + P(X = 4) + P(X = 5) = 0.11 + 0.04 + 0.08 = 0.23
$$

## 4. (blue, баллы: 1) Предскажи, сколько задач решит кандидат с вероятностью 90% или больше. Найди выполняющий это требование самый короткий интервал $[a;b]$ с включёнными краями. Иначе говоря, найди целые числа $a$ и $b$, чтобы с вероятностью 90% или больше кандидат решил от $a$ до $b$ задач включительно.

Если от 0 до 5 задач включительно, то вероятность 0.23 + 0.24 + 0.12 + 0.11 + 0.04 + 0.08 = 0.82

Если от 0 до 6 задач включительно, то вероятность 0.23 + 0.24 + 0.12 + 0.11 + 0.04 + 0.08 + 0.10 = 0.92

Значит, с вероятностью 90% кандидат решит от 0 до 6 задач включительно.



## 5. (blue, баллы: 0.7) Найди матожидание $EX$ и проинтерпретируй его.

In [1]:
import pandas as pd

df = pd.DataFrame({
    'x': range(9),
    'p': [0.23, 0.24, 0.12, 0.11, 0.04, 0.08, 0.10, 0.05, 0.03]
})
df.set_index('x').transpose()

x     0     1     2     3     4     5    6     7     8
p  0.23  0.24  0.12  0.11  0.04  0.08  0.1  0.05  0.03

In [2]:
ex = (df['x'] * df['p']).sum()
ex

np.float64(2.5600000000000005)

Матожидание: 2,56.

Это означает, что в среднем кандидат при большом количестве повторений мы ожидаем, что кондидат решает 2,56 задачи на первом этапе.



## 6. (blue, баллы: 0.7) Найди дисперсию $VX$ и не интерпретируй её.

In [3]:
vx = (df['x'] ** 2 * df['p']).sum() - ex ** 2
vx

np.float64(5.766399999999999)



## 7. (blue, баллы: 1) Измени порог прохождения во второй этап с $5$ задач на $k$ задач.
Найди функцию отсева $G(k)$, показывающую вероятность отсева кандидата, то есть заполни табличку:

<table>
<caption>
Вероятность отсева кандидата, если для прохождения требуется решить не менее $k$ задач.
</caption>
  <thead>
    <tr>
      <th>$k$</th>
      <th>$0$</th>
      <th>$1$</th>
      <th>$2$</th>
      <th>$3$</th>
      <th>$4$</th>
      <th>$5$</th>
      <th>$6$</th>
      <th>$7$</th>
      <th>$8$</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>$G(k)$</b></td>
      <th>$0.??$</th>
      <th>$0.??$</th>
      <th>$0.??$</th>
      <th>$0.??$</th>
      <th>$0.??$</th>
      <th>$0.??$</th>
      <th>$0.??$</th>
      <th>$0.??$</th>
      <th>$0.??$</th>
    </tr>
  </tbody>
</table>


In [8]:
G = []
for k in range(9):
  gk = df[df['x'] < k]['p'].sum()
  G.append(gk)

df1 = pd.DataFrame({
    'k': range(9),
    'G(k)': G
})

df1.set_index('k').transpose()

k       0     1     2     3    4     5     6     7     8
G(k)  0.0  0.23  0.47  0.59  0.7  0.74  0.82  0.92  0.97



## 8. (blue, баллы: 0.5) Есть ли разница между кумулятивной функцией распределения $CDF(k)$ и функцией $G(k)$, найденной выше? Если да, то в чём?

CDF(k) - вероятность того, что кандидат решит не более k задач.

G(k) - вероятность того, что кандидат решит менее k задач.


# Задача 2. Освой накопленную и обратную функции распределения


Студент на экзамене может получить в качестве оценки $X$ целое число от 2 до 5 включительно. Вероятностная модель оценки задана кумулятивной функцией распределения $CDF(k)=P(X\leqslant k)$.

<table>
<caption>
Кумулятивная функция распределения $X$ — оценки студента
</caption>
  <thead>
    <tr>
      <th>k</th>
      <th>$2$</th>
      <th>$3$</th>
      <th>$4$</th>
      <th>$5$</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>CDF(k)</td>
      <th>$0.15$</th>
      <th>$0.35$</th>
      <th>$0.70$</th>
      <th>$1.00$</th>
    </tr>
  </tbody>
</table>


##1. (blue, баллы: 0.3) Найди вероятность получить оценку не выше 4: $P(X\leqslant 4)$.

CDF(4) = 0.7

##2. (blue, баллы: 0.4) Найди $P(X < 4)$.

CDF(3) = 0.35

##3. (blue, баллы: 0.4) Найди $P(3 \leqslant X \leqslant 4)$.

CDF(4) - CDF(2) = 0.7 - 0.15 = 0.55

##4. (blue, баллы: 0.4) Найди вероятность получить оценку выше $3$.

$P(X > 3) = 1 - P(X \leqslant 3) = 1 - CDF(3) = 1 - 0.35 = 0.65

##5. (blue, баллы: 0.4) Найди значение обратной функции распределения $PPF(0.7)$ и проинтерпретируй его.

PPF(0.7) = 4

Это означает, что с вероятностью 0.7 студент получит оценку 4 или ниже.

##6. (blue, баллы: 0.4) Найди $PPF(0.3)$ и проинтерпретируй его.

PPF(0.3) = 2

Это означает, что с вероятностью 0.3 студент получит оценку 2 или ниже.


##7. (blue, баллы: 0.9) Задай распределение $X$ некумулятивной функцией вероятности $PMF(k)=P(X=k)$.

  <table>
<caption>
Распределение $X$ — оценки студента.
</caption>
  <thead>
    <tr>
      <th>$k$</th>
      <th>$2$</th>
      <th>$3$</th>
      <th>$4$</th>
      <th>$5$</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>PMF(k)</b></td>
      <th>$ 0.15 $</th>
      <th>$ 0.20 $</th>
      <th>$ 0.35 $</th>
      <th>$ 0.30 $</th>
    </tr>
  </tbody>
</table>




# Задача 3. Переведи задачу в вероятностную модель


Компания предлагает пешеходам страховой продукт «АнтиДТПин», который действует 1 год. Каждый клиент платит взнос 1100 рублей в начале года. В случае попадания в ДТП клиент получает выплату по страховке: 80 тысяч рублей в случае ДТП с мотоциклом или легковым автомобилем и 120 тысяч рублей в любом ином случае ДТП (с грузовым автомобилем, автобусом или спецтехникой: трактором, подъёмным краном и прочим).

Страховая компания считает, что 99.9% клиентов не попадут в ДТП. Шанс на ДТП с мотоциклом компания оценивает как 1 из 10000, шанс попадания в ДТП с легковым автомобилем компания оценивает как 3 из 10000.


##1. (blue, баллы: 1) Введи обозначение для случайной величины, показывающей прибыль компании от конкретного клиента, и построй вероятностную модель, которая описывает такую случайную величину. Это можно сделать несколькими способами, выбирай любой.

##2. (blue, баллы: 0.7) Найди матожидание прибыли компании от конкретного клиента.

**Решение**

# Задача 4. Найди характеристики случайной величины после линейного преобразования


Продолжение задачи 1 (можно и нужно пользоваться найденными в ней $EX$ и $VX$).

За каждый правильный ответ кандидат набирает 10 баллов, а за неправильный/отсутствующий — теряет 4 балла. Обозначим за $Y$ количество баллов, набранных кандидатом.


##1. (red, баллы: 0.3) Объясни, почему $Y=14\cdot X - 32$.

##2. (red, баллы: 0.3) Найди матожидание $EY$, используя найденное ранее матожидание $EX$.

##3. (red, баллы: 0.3) Найди дисперсию $VY$, используя найденную ранее дисперсию $VX$.

##4. (red, баллы: 0.8) Найди распределение $Y$. Найдите $EY$ и $VY$ напрямую из распределения $Y$.

**Решение**

**1**

$Y = 10X - (8 - X) * 4 = 10X - 32 + 4X = 14X - 32$

**2**

$EY = 14EX - 32 = 14 * 2.56 - 32 = 3.84$

**3**

$VY = 14^2 * VX = 14^2 * 5,77 = 1130.92$

**4**

<table>
<caption>
Распределение $Y$.
</caption>
  <thead>
    <tr>
      <th>$y$</th>
      <th>$-32$</th>
      <th>$-18$</th>
      <th>$-4$</th>
      <th>$10$</th>
      <th>$24$</th>
      <th>$38$</th>
      <th>$52$</th>
      <th>$66$</th>
      <th>$80$</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>$P(Y=y)$</b></td>
      <th>$0.23$</th>
      <th>$0.24$</th>
      <th>$0.12$</th>
      <th>$0.11$</th>
      <th>$0.04$</th>
      <th>$0.08$</th>
      <th>$0.10$</th>
      <th>$0.05$</th>
      <th>$0.03$</th>
    </tr>
  </tbody>
</table>



In [13]:
import pandas as pd

df = pd.DataFrame({
    'y': [-32, -18, -4, 10, 24, 38, 52, 66, 80],
    'p': [0.23, 0.24, 0.12, 0.11, 0.04, 0.08, 0.10, 0.05, 0.03]
})
df.set_index('y').transpose()

y   -32   -18   -4     10    24    38   52    66    80
p  0.23  0.24  0.12  0.11  0.04  0.08  0.1  0.05  0.03

In [14]:
ey = (df['y'] * df['p']).sum()
ey

np.float64(3.840000000000001)

In [16]:
vy = (df['y'] ** 2 * df['p']).sum() - ey ** 2
vy

np.float64(1130.2144)

# Задача 5. Найди характеристики суммы двух независимых случайных величин

Продолжение задачи 1 (можно и нужно пользоваться найденными в ней $EX$ и $VX$).

Два кандидата независимо проходят первый этап. Количества правильно решённых каждым из них задач обозначим за $X_1$ и $X_2$ соответственно. Распределение у $X_1$ и $X_2$ одинаковые, табличка распределения представлена в задаче 1.


##1. (red, баллы: 0.3) Найди $E(X_1 + X_2)$ и объясни смысл найденного числа.

$E(X_1 + X_2) = EX_1 + EX_2 = 2 * 2.56 = 5.12$

Это означает, что в среднем два кандидата в сумме решают 5.12 задачи на первом этапе.

##2. (red, баллы: 0.3) Найди $V(X_1 + X_2)$.

$V(X_1 + X_2) = VX_1 + VX_2 = 5.77 + 5.77 = 11.54$


##3. (red, баллы: 0.3) Верно ли, что величины $X_1$ и $X_2$ одинаково распределены, но не равны?

Ответ: да, величины одинаково распределены, но не равны.

##4. (black, баллы: 0.7) Найди совместное распределение $X_1 + X_2$ и посчитай $E(X_1+X_2)$ и $V(X_1 + X_2)$. Проверь, что они совпадают с ответами пунктов 1 и 2.


In [23]:
p = [0.23, 0.24, 0.12, 0.11, 0.04, 0.08, 0.10, 0.05, 0.03]

df = pd.DataFrame({
    'x':[x1 + x2 for x1 in range(9) for x2 in range(9)],
    'p': [p[x1] * p[x2] for x1 in range(9) for x2 in range(9)]
})

df = df.groupby('x').sum().reset_index()

df.set_index('x').transpose()

x      0       1       2       3       4       5       6      7      8   \
p  0.0529  0.1104  0.1128  0.1082  0.0856  0.0824  0.1061  0.099  0.081   

x      9       10      11      12      13      14     15      16  
p  0.0548  0.0326  0.0266  0.0204  0.0148  0.0085  0.003  0.0009

In [27]:
ey = (df['x'] * df['p']).sum()
ey

np.float64(5.12)

In [28]:
vy = (df['x'] ** 2 * df['p']).sum() - ey ** 2
vy

np.float64(11.532800000000005)

Другие два кандидата поступают нечестно. Один из них решает задачи, а другой у него списывает. В результате их решения совпадают. Количества правильно решённых каждым из них задач обозначим за $X_3$  и  $X_4$ соответственно.

##5. (red, баллы: 0.3) Найди $E(X_3 + X_4)$ и объясни смысл найденного числа.

##6. (red, баллы: 0.3) Найди $V(X_3 + X_4)$.

##7. (red, баллы: 0.3) Верно ли, что величины $X_3$ и $X_4$ одинаково распределены, но не равны?

##8. (black, баллы: 0.7) Найди совместное распределение $X_3 + X_4$ и посчитай $E(X_3+X_4)$ и $V(X_3 + X_4)$. Проверь, что они совпадают с ответами пунктов 5 и 6.


**Решение**

# Задача 6. Найди распределение нелинейной функции от случайной величины



Продолжение задачи 1.

Правильно решённые задачи конвертируются в баллы $Y$ кандидата по следующей нелинейной формуле $Y=\sqrt{1+X}$. Например, кандидат, решивший $X=1$ задачу, набирает $\sqrt{2}\approx 1.414$ баллов (баллы могут быть дробными).

Чтобы набрать $Y=1$ балл, достаточно решить $X=0$.

$Y=2$ балла получит кандидат, решивший $X=3$ задачи.

$Y=3$ балла получит тот, кто решил все $X=8$ задач.

Можно заметить, что каждый дополнительный балл требует всё больше усилий: первый балл дают просто так, второй — за 3 задачи, а третий — за дополнительные 5 задач.

Такая нелинейная шкала позволяет спрятать выдающихся кандидатов: решившие намного больше задач будут иметь ненамного больше баллов.

##1. (red, баллы: 0.5) Найди распределение $Y=\sqrt{1+X}$.

##2. (red, баллы: 0.5) Найди $E(\sqrt{1+X})$ и сравни с $\sqrt{1+EX}$. Объясни смысл каждого из чисел в контексте ситуации.

##3. (red, баллы: 0.5) Найди $V(\sqrt{1+X})$ и сравни с $\sqrt{1+VX}$.



**Решение**

# Задача 7. Найди характеристики среднего и суммы случайных величин


(black) Продолжение задачи 1.

Кандидаты независимо проходят первый этап. Их $n$ человек. Количества правильно решённых каждым из них задач обозначим за  $X_1$, $X_2$, $\dots$, $X_n$. Вероятностная модель для каждого кандидатов описывается таблицей в задаче 1.


##1. (black, баллы: 0.3) Найди $E\left(\sum\limits_{i=1}^{n} X_i\right)$.

$$
E\left(\sum\limits_{i=1}^{n} X_i\right) = \sum\limits_{i=1}^{n} EX_i = n * EX = 2.56n
$$

##2. (black, баллы: 0.3) Найди $E\left(\overline{X}\right)$ и объясните смысл найденного числа и смысловую разницу с $\overline{X}$.

$$
E\left(\overline{X}\right) = EX = 2.56
$$

Смысл: ожидаемое значение среднего количества решенных задач на первом этапе.

##3. (black, баллы: 0.3) Найди $V\left(\sum\limits_{i=1}^{n} X_i\right)$.

$$
V\left(\sum\limits_{i=1}^{n} X_i\right) = \sum\limits_{i=1}^{n} VX_i = n * VX = 5.77n
$$


##4. (black, баллы: 0.3) Найди $V\left(\overline{X}\right)$.

$$
V\left(\overline{X}\right) = \frac{VX}{n} = \frac{5.77}{n}
$$

##5. (black, баллы: 0.3) Верно ли, что величины $X_1$, $X_2$, $\dots$, $X_n$ независимые, одинаково распределенные, но не совпадающие?

Ответ: да, величины независимые, одинаково распределенные, но не совпадающие.



**Решение**

# Задача 8. Освой двумерное дискретное распределение


В этой задаче придётся работать с абстрактной моделью. Случайные величины $X$ и $Y$ не наделены конкретным прикладным смыслом, дана лишь модель их совместного распределения.



<table>
<caption>
Совместное распределение $X$ и $Y$.
</caption>
  <thead>
    <tr>
      <th >..............</th>
      <th>$X=-1$</th>
      <th>$X=0$</th>
      <th>$X=1$</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>$Y=1$</td>
      <th>$0.2$</th>
      <th>$0.0$</th>
      <th>$0.2$</th>
    </tr>
    <tr>
      <td>$Y=0$</td>
      <th>$0.0$</th>
      <th>$0.6$</th>
      <th>$0.0$</th>
    </tr>
  </tbody>
</table>

1. (black, баллы: 0.1) Запиши распределение в виде одномерной таблицы.

  <table>
<caption>
Совместное распределение.
</caption>
  <thead>
    <tr>
      <th>$X$</th>
      <th>$Y$</th>
      <th>Probability</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>$X=-1$,</td>
      <th>$Y=1$</th>
      <th>$0.??$</th>
    </tr>
    <tr>
      <td>$\dots$</td>
      <th>$\dots$</th>
      <th>$\dots$</th>
    </tr>
  </tbody>
</table>

2. (black, баллы: 0.5) Проверь математически по определению, являются ли $X$ и $Y$ независимыми.

3. (black, баллы: 0.4) Верно ли, что если известно значение случайной величины $X$, то можно однозначно определить значение случайной величины $Y$? Есть ли связь между случайными величинами по смыслу? Напиши квадратичное равенство, связывающее эти величины.

4. (black, баллы: 0.5) Найди распределение $X$.

  <table>
<caption>
Распределение $X$
</caption>
  <thead>
    <tr>
      <th>$x$</th>
      <th>.........</th>
      <th>.........</th>
      <th>.........</th>
      <th>.........</th>
      <th>.........</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>$P(X=x)$</b></td>
      <th>$0.??$</th>
      <th>$0.??$</th>
      <th>$0.??$</th>
      <th>$0.??$</th>
      <th>$0.??$</th>
    </tr>
  </tbody>
</table>

5. (black, баллы: 0.5) Найди распределение $Y$.

  <table>
<caption>
Распределение $Y$
</caption>
  <thead>
    <tr>
      <th>$y$</th>
      <th>.........</th>
      <th>.........</th>
      <th>.........</th>
      <th>.........</th>
      <th>.........</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>$P(Y=y)$</b></td>
      <th>$0.??$</th>
      <th>$0.??$</th>
      <th>$0.??$</th>
      <th>$0.??$</th>
      <th>$0.??$</th>
    </tr>
  </tbody>
</table>

6. (black, баллы: 0.6) Найди $EX$ и $EY$.

7. (black, баллы: 0.8) Найди $E(XY)$, $cov(X,Y)=E(XY)-EX\cdot EY$ и корреляцию $corr(X,Y)=\frac{cov(X,Y)}{\sqrt{VX}\sqrt{VY}}$.

8. (black, баллы: 0.4) Может ли быть, что случайные величины являются нескоррелированными (корреляция равна нулю), но зависимыми?

9. (black, баллы: 0.4) Известно, что корреляция показывает направление и силу линейной связи. Почему в этом примере отсутствие линейной связи не означает независимость? Какая связь между $X$ и $Y$ всё же присутствует?


**Решение**

# Задача 9. Переведи прикладную задачу на язык модели




(black) На сайте телешоу зарегистрированным пользователям предлагается проголосовать за одного из трёх финалистов шоу: Akro, Berro и Cirto. Пусть в генеральной совокупности 40% поддерживают Akro, 35% поддерживают Berro и 25% поддерживают Cirto.

Проголосовали $n=1000$ пользователей. Будем считать, что общее число пользователей настолько велико, что выборку можно рассматривать как случайную выборку с возвращением из генеральной совокупности.
Обозначим за $A$ количество проголосовавших за Akro, $B$ — за Berro и $C$ — за Cirto.

##1. (black, баллы: 0.4) Докажи, что случайные величины $A$ и $B$ зависимы.

##2. (black, баллы: 0.4) Кратко объясни, почему совместное распределение $A$ и $B$ не получится описать небольшой понятной табличкой, из которой была бы видна сила связи между ними.

##3. (black, баллы: 0.1) Пусть $X_i$ — случайная величина, которая равна 1, если $i$-й человек проголосовал за $A$, и 0 в остальных случаях. Пусть $Y_i$ — случайная величина, которая равна 1, если $i$-й человек проголосовал за $B$, и 0 в остальных случаях. Установи связь между $A$ и $X_1,X_2,\dots,X_n$. Аналогично выпиши связь между $B$ и $Y_1,Y_2,\dots,Y_n$.

##4. (black, баллы: 0.4) Построй совместное распределение $X_1,Y_1$. Найди $E(X_i\cdot Y_i)$, $E(X_i)$, $E(Y_i)$

##5. (black, баллы: 0.3) Используя факт из теории вероятностей, что для независимых случайных величин матожидание произведения равно произведению матожиданий, найди $E(X_i\cdot Y_j)$, если $i\ne j$.

##6. (black, баллы: 0.4) Найди ковариацию $cov(A,B)=E(A\cdot B)-EA\cdot EB$, используя разложение и промежуточные вычисления из предыдущих пунктов.

##7. (black, баллы: 0.4) Найди корреляцию $corr(A,B)=\frac{cov(A,B)}{\sqrt{VA}\sqrt{VB}}$.

##8. (black, баллы: 0.5) Используя то, что корреляция показывает силу и направление линейной связи, дай интуитивное объяснение, почему у корреляции, найденной в предыдущем пункте, получился именно такой знак.

**Решение**

**1**

$$
P(A \cdot B) = 0
$$

$$
P(A) = 0.4
$$

$$
P(B) = 0.35
$$

$$
P(A \cdot B) \ne P(A) \cdot P(B)
$$

Значит, что зависимы. Так как можно проголосовать только за одного.

**2**

Потому что события $A$ и $B$ не могут произойти одновременно.

**3**

$$
A = \sum\limits_{i=1}^{n} X_i
$$

$$
B = \sum\limits_{i=1}^{n} Y_i
$$

**4**

$$
E(X_i) = 0.4
$$

$$
E(Y_i) = 0.35
$$

$$
E(X_i \cdot Y_i) = 0
$$

<table>
<caption>
Совместное распределение $X_1$ и $Y_1$.
</caption>
  <thead>
    <tr>
      <th>$X_1$</th>
      <th>$Y_1$</th>
      <th>Probability</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>$0$</td>
      <th>$0$</th>
      <th>$0.25$</th>
    </tr>
    <tr>
      <td>$1$</td>
      <th>$0$</th>
      <th>$0.4$</th>
    </tr>
    <tr>
      <td>$0$</td>
      <th>$1$</th>
      <th>$0.35$</th>
    </tr>
    <tr>
      <td>$1$</td>
      <th>$1$</th>
      <th>$0$</th>
    </tr>
  </tbody>


  
**5**

$$
E(X_i \cdot Y_j) = E(X_i) \cdot E(Y_j) = 0.4 \cdot 0.35 = 0.14
$$

**6**

$$
cov(A,B) = E(A \cdot B) - E(A) \cdot E(B) = 0.14 - 0.4 \cdot 0.35 = 0
$$
